src/main/java/lib/Utils.java

```diff
@@ -60,4 +60,11 @@ public static Pose2d calcOptimalPose(List<Pose2d> points, Pose2d robotPose) {
                                 point -> getDistanceFromPoint(point.getTranslation(), robotPose)))
                 .orElse(null);
     }
+
+    public static Rotation2d calculateOptimalRotation(
```

Comment:
The name is a bit excessive, there is nothing "optimal" in this method.

Reply:
ca4e6a4

Reply:
74eea28